package.json

```diff
@@ -1,22 +1,31 @@
 {
   "name": "@tduniec/backstage-timesaver-plugin",
-  "version": "1.0.0",
+  "version": "1.0.1",
+  "description": "This plugin provides an implementation of charts and statistics related to your time savings that are coming from usage of your templates.",
+  "private": true,
   "engines": {
     "node": "18 || 20"
   },
   "workspaces": {
-  "packages": [
-    "plugins/*"
-  ]},
+    "packages": [
+      "plugins/*"
+    ]
+  },
   "repository": "https://github.com/tduniec/backstage-timesaver-plugin.git",
   "author": "tduniec <tomekduniec@gmail.com>",
   "license": "Apache-2.0",
   "scripts": {
     "build": "yarn workspaces run build",
     "clean": "yarn workspaces run clean",
+    "tsc": "tsc",
     "test": "yarn workspaces run test",
```

Comment:
Hi @tduniec. I notice you're using jest tool directly instead of the backstage-cli. Backstage has already setup a bunch of configurations and things to look for when doing tests, same goes for linting. What do you think about using their tools to get this plugin reviewed and approved quicker? Maybe not for this PR to get it merged ASAP but for future usage.